bindings/go/constructor/construct_test.go

```diff
@@ -0,0 +1,81 @@
+package constructor
```

Comment:
Do we need any test that contains resources and sources?

Reply:
sure, although I didnt find them necessary. I can add one for good measure 👍

Reply:
added